source/process/software-requirements.rst

```diff
@@ -14,7 +14,7 @@ Desktop Apps
 
     "Windows", "Supported versions by Microsoft - `reference <https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions>`__"
     "Mac", "Supported versions by Apple - `reference <https://en.wikipedia.org/wiki/MacOS_version_history>`__"
-    "Linux", "Fixed to Ubuntu LTS releases 14.04 or later"
+    "Linux", "Fixed to Ubuntu LTS releases 16.04 or later"
 
 PC Web
 ---------------------------------
```

Comment:
Do we need to make any changes to our process guidelines regarding Safari version support, given we're keeping it at v12+ instead of changing to v13+?

Reply:
We don't have official guides for them yet, but they should work similar if not the same unless the maintainers of said distros did some major rework between versions. Last time we looked at it it was the mysql version that was blocking updating the documentation.

We could do `Latest and next-to-latest versions`  or use Chen's proposal `Safari version available in the minimum supported iOS version `